metrics.yaml

```diff
@@ -105,7 +105,7 @@ parameters:
 -
   description: 'Specify version for metrics components; e.g. for "openshift/origin-metrics-deployer:latest", set version "latest"'
   name: IMAGE_VERSION
-  value: "3.4.0"
+  value: "v3.5"
```

Comment:
latest is what we wish i believe.  I was just pointing out the diff and if it matter.  erroring on the side of consistency